src/Hydra/Serializer/DocumentationNormalizer.php

```diff
@@ -396,13 +396,13 @@ private function getClasses(array $entrypointProperties, array $classes): array
                 [
                     '@type' => 'hydra:SupportedProperty',
                     'hydra:property' => [
-                        '@id' => '#ConstraintViolationList/violation',
-                        '@type' => 'rdf:Property',
-                        'rdfs:label' => 'violation',
+                        '@id' => '#ConstraintViolationList/violations',
+                        '@type' => 'hydra:Link',
```

Comment:
@dunglas you are right, I can check if the targeted range is a supported class or not to guess if the property is a relation, and no more depend on `@type` of the property. It works fine. Tell me when you want I update my PR to keep `rdf:Property`.